nomt/src/rw_pass_cell.rs

```diff
@@ -260,15 +262,6 @@ impl WritePass<UniversalRegion> {
     }
 }
 
-impl<R> Drop for WritePass<R> {
```

Comment:
Oh nice, I think this allows us to remove the pesky `clone`-s above since now we can destruct `self: WritePass`.